easybuild/easyconfigs/q/qpth/qpth-0.0.13_20190626-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'qpth'
+version = '0.0.13_20190626'
+# this commit has patches on top of version 0.0.13 to make it work with the latest version of Pytorch (1.1.0)
+# see https://github.com/locuslab/qpth/issues/23
+# commit = '411c64f'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://locuslab.github.io/qpth/'
+description = """ A fast and differentiable QP solver for PyTorch. """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('SciPy-bundle', '2019.03'),
+    ('PyTorch', '1.1.0', versionsuffix),
+    ('CVXPY', '1.0.24'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('block', '0.0.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/block'],
+        'checksums': ['8de2c9e37959f3851f1e7b7475ab797d1de4ec3e825352cdde0fcd01ec1c7c67'],
+    }),
+    (name, version, {
+        'source_tmpl': '411c64f.tar.gz',
```

Comment:
This is going to be easy to overlook when updating this easyconfig...

Better use `'source_tmpl': '%s.tar.gz' % local_commit,` here, and uncomment the `commit =` (and rename to `local_commit = `)?